### PR TITLE
Migrate to Codefresh from Travis

### PR DESCRIPTION
**Pull request by maximmi**
## What
* Codefresh support

## Why
* going to migrate from TravisCI to Codefresh

### CODE REVIEW

File: codefresh.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    image: alpine
+    commands:
+    - cf_export TESTING_CLOUDPOSSE_CO_VERSION=tests
+
+  test:
+    title: Terraform module test
+    image: r.cfcr.io/cloudposse/testing.cloudposse.co:${{TESTING_CLOUDPOSSE_CO_VERSION}}
+    working_directory: /
+    commands:
+    - cd /codefresh/volume/${{CF_REPO_NAME}}
```
Comment (goruha) [inline codefresh.yml]: why we need this?

File: codefresh.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#    - make terraform/install
```
Comment (goruha) [inline codefresh.yml]: Remove comments

File: codefresh.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+version: '1.0'
+
+steps:
+  test:
+    title: Terraform module test
```
Comment (osterman) [inline codefresh.yml]:
```suggestion
    title: Terraform Module Integration Tests
```

**Comment by osterman** [comment]
Should we remove travis in this PR?

**Comment by aknysh** [comment]
@maximmi  see example here on how to add Codefresh badge
https://github.com/cloudposse/terraform-root-modules/blob/master/README.yaml#L38

(I tried to create a new repo on Codefresh for https://github.com/cloudposse/terraform-aws-mq-broker, but got an error "Failed to create Service". Need to investigate why it's happening)

File: Makefile [hunk 4]
```diff
@@ -3,7 +3,7 @@ SHELL := /bin/bash
 # List of targets the `readme` target should call before generating the readme
 export README_DEPS ?= docs/targets.md docs/terraform.md
 
--include $(shell curl -sSL -o .build-harness "https://git.io/build-harness"; echo .build-harness)
+-include $(shell curl -k -sSL -o .build-harness "https://git.io/build-harness"; echo .build-harness)
```
Comment (goruha) [inline Makefile]:
```suggestion
-include $(shell curl -sSL -o .build-harness "https://git.io/build-harness"; echo .build-harness)
```

**Pull request closed without merging**